ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v2/checkboxgroup/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please update component readme to include the exact changes, also update the BEM as per the new HTML in the component readme